udata_croquemort/checker.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+import dateutil.parser
+
+from .utils import check_url, UnreachableLinkChecker
+
+log = logging.getLogger(__name__)
+
+
+class CroquemortLinkChecker(object):
+    """Croquemort link checker implementation.
+
+    The main interface is the `check` method.
+    """
+
+    def _format_response(self, response):
+        status = response.get('final-status-code')
+        try:
+            status = int(status)
+            return {
+                'check:url': response.get('checked-url'),
```

Comment:
Nice namespacing, do we expect anything that is not prefixed by `check:`?

Reply:
Nope, all `check:`